### ISSUE TITLE
django.db.models.get_model deprecated in Django 1.9

### ISSUE THREAD

**Issue by macolo** [jrief/djangocms-cascade#135]
I get the following error on Django 1.9 after installing djangocms-cascade:

```
  File "/Users/mario/.virtualenvs/what-digital-djcms/src/djangocms-cascade/cmsplugin_cascade/mixins.py", line 8, in <module>
    from django.db.models import get_model
ImportError: cannot import name 'get_model'
```
I hear that the get_model alias is deprecated in Django 1.9 and can be replaced by `django.apps.apps.get_model`

**Comment by rfleschenberg** [comment]
Cascade is not compatible with Django 1.9 yet. For the time being, please use Django 1.8. We hope that we can fix this soon, but we are also rather busy. Pull requests that improve Django 1.9 compatibility are very welcome ;)

**Comment by rfleschenberg** [comment]
Fixed by PR #146.
